mautic/mautic
# Fix tag name cannot be empty and throw validation error

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | ❌ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ✅  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


The tag name should not be empty. This PR takes care of this issue.

#### Steps to reproduce:
1. Go to tags from left side bar menu
2. Add New
3. Click Save and Close
4. Observe empty tag getting saved

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. After PR empty tags should not be saved




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11120"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
